Makefile

```diff
@@ -71,6 +71,25 @@ image-%:
 	$(eval img := $(subst image-,,$@))
 	gcloud builds submit --config './images/$(img)/cloudbuild.yaml' './images/$(img)'
 
+RUNTIME := podman
+LOCALIMAGE_NAME := k8s-cloud-builder
+
+# Build a local image to use the tools of this repository on non
+# Debian/Ubuntu/Fedora distributions
```

Comment:
If you'd add that comment after the target name, something like
```makefile
local-image-build: ## Build a local image to use the tools of this repository on non Debian/Ubuntu/Fedora distributions
```
it would show nicely when you run `make help` (or `make` without a target even, IIRC).

Reply:
Alright, fixed that as well